packages/shared/components/modals/ProfileActions.svelte

```diff
@@ -23,9 +26,30 @@
     const handleLogoutClick = async (): Promise<void> => {
```

Comment:
Can we change this to the syntax below (regular function instead of arrow function)? It will be the official way to write functions in the future. For now the rule of thumb is that within one file all the functions should be written in the same way.